### ISSUE TITLE
Editor sometimes creates invalid HTML

### ISSUE THREAD

**Issue by RandomlyKnighted** [steveathon/bootstrap-wysiwyg#70]
Originally documented at madskristensen/MiniBlog#153

In the original issue, they found that the editor will at times create invalid HTML. In my blog (that uses the Miniblog engine) I had a file that had multiple missing end tags.

I'm not sure how we might go about correcting this issue but it is something that we should definitely look into.

**Comment by waynezwf** [comment]
yes! if you change the color of text by that input box, sometimes the the editor will be create a span tag!
